packages/static-build/src/index.ts

```diff
@@ -442,34 +463,11 @@ export const build: BuildV2 = async ({
       } else if (typeof installCommand === 'string') {
         if (installCommand.trim()) {
           console.log(`Running "install" command: \`${installCommand}\`...`);
-          const { cliType, lockfileVersion } = await scanParentDirs(
-            entrypointDir
-          );
```

Comment:
We used to only scan when a custom install command was used. Are you sure we need to scan above too?